data/adas/read_radiation.py

```diff
@@ -0,0 +1,239 @@
+import os
```

Comment:
this file has barely any comments, and no header describing its purpose or what it does. Please at some description near the top